src/Outbox.php

```diff
@@ -0,0 +1,135 @@
+<?php
+declare(strict_types=1);
+
+namespace atk4\outbox;
+
+use atk4\core\AppScopeTrait;
+use atk4\core\DIContainerTrait;
+use atk4\core\Exception;
+use atk4\core\FactoryTrait;
+use atk4\core\InitializerTrait;
+use atk4\data\Persistence;
+use atk4\outbox\Model\Mail;
+
+class Outbox
+{
+    use AppScopeTrait;
+    use InitializerTrait {
+        init as _init;
+    }
+    use DIContainerTrait;
+    use FactoryTrait;
+
+    /**
+     * Mailer.
+     *
+     * @var MailerInterface
+     */
+    private $mailer;
```

Comment:
please use `protected` instead of `private` properties so this class can be extended more easily